cf-platform-eng/aws-broker-boshrelease
# [#117512347] RDS service broker bosh-release

## What

This is a forked release of aws-brokers. There were three of them here:
- rds broker
- sqs broker
- cloudformation broker

This PR removes sqs and cloudformation leaving only RDS broker that we care about. Also it replaces references to `cf-platform-eng/rds-broker` with our fork of rds-broker. 
# How to test

You can create DEV version of this release and upload it to your bosh instance. For easy testing I recommend:

```
git clone https://github.com/cloudfoundry/bosh-lite.git
cd bosh-lite
vagrant up
bosh target 192.168.50.4
echo admin | bosh login admin
```

and next inside  `aws-broker-boshrelease`:

```
 ./update
bosh create release --force
bosh upload release
```

You should see sth like that after executing `bosh releases`:

```
RSA 1024 bit CA certificates are loaded due to old openssl compatibility
Acting as user 'admin' on 'Bosh Lite Director'

+------------+----------+-------------+
| Name       | Versions | Commit Hash |
+------------+----------+-------------+
| aws-broker | 2+dev.3  | 00971cc2    |
|            | 2+dev.4  | 00971cc2    |
|            | 2+dev.5  | 00971cc2    |
|            | 2+dev.7  | fb3cf164+   |
+------------+----------+-------------+
(+) Uncommitted changes
```
# Who can test

Not @timmow or @combor
